### PR TITLE
feat(dockerfiles): add ci job runtimes and apps images

### PR DESCRIPTION
**Pull request by wuhuizuo**
- ci runtime image: bazel
- ci app image: ats

Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### CODE REVIEW

File: dockerfiles/WIP_ci_apps/ats/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Ref: https://github.com/sqawasmi/trafficserver-docker/blob/master/Dockerfile
+FROM debian:11.5
+
+LABEL org.opencontainers.image.authors=" wuhui.zuo@pingcap.com"
```
Comment (lijie0123) [inline dockerfiles/WIP_ci_apps/ats/Dockerfile:4]: remove space before you name

Comment (wuhuizuo) [inline dockerfiles/WIP_ci_apps/ats/Dockerfile:4]:
```suggestion
LABEL org.opencontainers.image.authors="wuhui.zuo@pingcap.com"
```

File: dockerfiles/WIP_ci_runtimes/bazel/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+ARG ARCH=amd64
```
Comment (wuhuizuo) [inline dockerfiles/WIP_ci_runtimes/bazel/Dockerfile:1]:
```suggestion
LABEL org.opencontainers.image.authors="wuhui.zuo@pingcap.com"

ARG ARCH=amd64
```

**Pull request merged**